.github/workflows/ci.yaml

```diff
@@ -425,6 +438,30 @@ jobs:
           config: "markdownlint-config.json"
           args: "README.md"
 
+  build_mirrord_on_release_branch:
```

Comment:
Why do we need a specific build step for the release branch? Don't we build mirrord on all branches anyway?

Reply:
- we don't explicitly build mirrord? not that i could find it in the steps of integration tests?
- building separately is faster, as we dont have to depend on heavy jobs and we can concurrently proceed with the ide e2e